[2025-09-18T05:44:27.994+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-18T05:44:28.004+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_yfinance_stock.fetch_and_upsert manual__2025-09-18T05:44:26.984218+00:00 [queued]>
[2025-09-18T05:44:28.007+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_yfinance_stock.fetch_and_upsert manual__2025-09-18T05:44:26.984218+00:00 [queued]>
[2025-09-18T05:44:28.008+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-09-18T05:44:28.012+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_and_upsert> on 2025-09-18 05:44:26.984218+00:00
[2025-09-18T05:44:28.015+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=7144) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-09-18T05:44:28.016+0000] {standard_task_runner.py:64} INFO - Started process 7146 to run task
[2025-09-18T05:44:28.017+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_yfinance_stock', 'fetch_and_upsert', 'manual__2025-09-18T05:44:26.984218+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/etl_yfinance_stock.py', '--cfg-path', '/tmp/tmp85z1cygi']
[2025-09-18T05:44:28.017+0000] {standard_task_runner.py:91} INFO - Job 60: Subtask fetch_and_upsert
[2025-09-18T05:44:28.032+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_yfinance_stock.fetch_and_upsert manual__2025-09-18T05:44:26.984218+00:00 [running]> on host 11e1994bdecf
[2025-09-18T05:44:28.208+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='dingyuyao' AIRFLOW_CTX_DAG_ID='etl_yfinance_stock' AIRFLOW_CTX_TASK_ID='fetch_and_upsert' AIRFLOW_CTX_EXECUTION_DATE='2025-09-18T05:44:26.984218+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-18T05:44:26.984218+00:00'
[2025-09-18T05:44:28.209+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-18T05:44:28.211+0000] {warnings.py:112} WARNING - /opt/***/dags/etl_yfinance_stock.py:36: FutureWarning: YF.download() has changed argument auto_adjust default to True
  df = yf.download(symbol, start = start_date.isoformat(), end = end_date.isoformat(), progress = False)

[2025-09-18T05:44:28.747+0000] {warnings.py:112} WARNING - /opt/***/dags/etl_yfinance_stock.py:36: FutureWarning: YF.download() has changed argument auto_adjust default to True
  df = yf.download(symbol, start = start_date.isoformat(), end = end_date.isoformat(), progress = False)

[2025-09-18T05:44:29.189+0000] {warnings.py:112} WARNING - /opt/***/dags/etl_yfinance_stock.py:36: FutureWarning: YF.download() has changed argument auto_adjust default to True
  df = yf.download(symbol, start = start_date.isoformat(), end = end_date.isoformat(), progress = False)

[2025-09-18T05:44:29.574+0000] {warnings.py:112} WARNING - /opt/***/dags/etl_yfinance_stock.py:36: FutureWarning: YF.download() has changed argument auto_adjust default to True
  df = yf.download(symbol, start = start_date.isoformat(), end = end_date.isoformat(), progress = False)

[2025-09-18T05:44:30.006+0000] {warnings.py:112} WARNING - /opt/***/dags/etl_yfinance_stock.py:36: FutureWarning: YF.download() has changed argument auto_adjust default to True
  df = yf.download(symbol, start = start_date.isoformat(), end = end_date.isoformat(), progress = False)

[2025-09-18T05:44:30.302+0000] {logging_mixin.py:188} INFO - [fetch_and_upsert] Removed 615 rows with NaN values
[2025-09-18T05:44:30.302+0000] {logging_mixin.py:188} INFO - [fetch_and_upsert] No valid data after cleaning NaN values.
[2025-09-18T05:44:30.303+0000] {python.py:237} INFO - Done. Returned value was: 0
[2025-09-18T05:44:30.303+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-18T05:44:30.314+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_yfinance_stock, task_id=fetch_and_upsert, run_id=manual__2025-09-18T05:44:26.984218+00:00, execution_date=20250918T054426, start_date=20250918T054428, end_date=20250918T054430
[2025-09-18T05:44:30.346+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-18T05:44:30.355+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-18T05:44:30.356+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
